### ISSUE TITLE
MultiPhotos - thumbs etc not displayed in Manager after update to 1.2.1

### ISSUE THREAD

**Issue by nick0** [modxcms/evolution#1221]
Hello,

I have recently updated a site from 1.0.15 to 1.2.1. The site is using PHP 7.1. and MultiPhotos 1.2.6 for per page photo galleries.

In doing so, the pre-existing MultiPhotos galleries have stopped displaying as the 3 fields per image (with an image thumbnails and title) per image in the Evo Manager - instead any MultiPhotos info is displayed in a textarea like so:

`[["assets/images/thumb_image1.jpg","assets/images/image1.jpg","image 1"],["assets/images/thumb_image2.jpg","assets/images/image2.jpg","image 2"],["assets/images/thumb_image3.jpg","assets/images/image3.jpg","image 3"]]`

The TV input type is originally textarea, which previously worked fine.

The thumbs do display as expected in the frontend (as clickable thumbs that enlarge in a lightbox), they just are not very user friendly in the Manager for the client with the raw code.

Any ideas on what I should tweak to get this working?

Thanks for any help.

**Comment by Deesen** [comment]
Without having used MultiPhotos myself yet, are there any Javascript-Errors in console?

**Comment by Fr3ddy7** [comment]
use multitv 2.0.8 )

**Comment by Fr3ddy7** [comment]
or put rule to plugin = 1,2,3

**Comment by nick0** [comment]
> Without having used MultiPhotos myself yet, are there any Javascript-Errors in console?

Thanks @Deesen. Hmmm, none that I could see. I remember seeing something like this with MultiTV ages ago and I think that was caused by jquery being included twice or a conflict.

> use multitv 2.0.8 )

Thanks @Fr3ddy7. That's my go to for galleries now, I just wanted to avoid having to recreate them all. Done now in MultiTV.

I am not sure what you mean by "or put rule to plugin = 1,2,3"

This is not resolved but closing now as I recreated the galleries in MultiTV and using that instead. Cheers
